ruby_event_store/lib/ruby_event_store/client.rb

```diff
@@ -216,6 +221,16 @@ def enrich_event_metadata(event)
       event.metadata[:timestamp] ||= clock.call
     end
 
+    def append_to_stream_serialized_events(serialized_events, stream_name:, expected_version:)
+      repository.append_to_stream(serialized_events, Stream.new(stream_name), ExpectedVersion.new(expected_version))
+    end
+
+    def prepare_events(events)
```

Comment:
This method name is suboptimal `enrich_events_metadata` or `enrich_events` would be better, I believe.